pkg/arvo/lib/pull-hook.hoon

```diff
@@ -1,17 +1,49 @@
+::  lib/pull-hook/hoon: helper for creating a push hook
+::  
+::   lib/pull-hook is a helper for automatically pulling data from a
+::   corresponding push-hook to a store. 
+::   
+::   ## Interfacing notes:
+::
+::   lib/push-hook provides the inner door with its own separate state.
```

Comment:
The phrasing here feels a bit confusing, but all it's really saying is that the inner agent's `+on-save` result gets threaded through to its `+on-load` transparently, right? Might not even be worth mentioning, if it just preserves expected behavior. But ymmv.